packages/modules/src/utils/Constants.ts

```diff
@@ -27,10 +27,10 @@ export const SESSION_MANAGER_MODULE_ADDRESSES_BY_VERSION = {
   V1_0_1: "0x000002FbFfedd9B33F4E7156F2DE8D48945E7489",
 };
 
-export const DEFAULT_BATCHED_SESSION_ROUTER_MODULE = "0x000008dA71757C0E1D83CE56c823e25Aa49bC058";
+export const DEFAULT_BATCHED_SESSION_ROUTER_MODULE = "0x00000D09967410f8C76752A104c9848b57ebba55";
 
 export const BATCHED_SESSION_ROUTER_MODULE_ADDRESSES_BY_VERSION = {
-  V1_0_0: "0x000008dA71757C0E1D83CE56c823e25Aa49bC058",
+  V1_0_0: "0x00000D09967410f8C76752A104c9848b57ebba55",
```

Comment:
Should we add this as new version or override it.

Reply:
override because we did vuln fix